Libraries/Components/SegmentedControlIOS/SegmentedControlIOS.ios.js

```diff
@@ -111,7 +106,14 @@ const SegmentedControlIOSWithRef = React.forwardRef(
     props: SegmentedControlIOSProps,
     forwardedRef: ?React.Ref<typeof RCTSegmentedControlNativeComponent>,
   ) => {
-    return <SegmentedControlIOS {...props} forwardedRef={forwardedRef} />;
+    return (
+      <SegmentedControlIOS
+        {...props}
+        enabled={props.enabled === undefined ? true : props.enabled}
```

Comment:
I think to be consistent with us replacing `defaultProps`, I'd lean towards doing this in `SegmentedControlIOS`. 

If you do that, you can also pull the props out on line 86 and use the default assignment `const {values = [], ....} = this.props`